projects/cashmere/src/lib/input/hc-form-field.component.scss

```diff
@@ -0,0 +1,121 @@
+@import '../sass/colors';
+
+$mat-form-field-default-infix-width: 120px;
+$line-height: 1.5;
+$label-font-scale: 1;
+$error-font-scale: 0.75;
+$infix-margin-top: 1em * $line-height * $label-font-scale;
+$prefix-suffix-icon-font-size: 125%;
+
+$infix-padding: 0.4375em;
+$error-margin-top: 0.4em / $error-font-scale;
+$wrapper-padding-bottom: ($error-margin-top + $line-height) * $error-font-scale;
+
+.hc-form-field {
+    display: inline-block;
+    position: relative;
+    text-align: left;
+
+    font-size: inherit;
+    font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-weight: 400;
+    line-height: $line-height;
+}
+
+.hc-form-field-wrapper {
+    position: relative;
+    border-top: $infix-margin-top solid transparent;
+    padding-bottom: $wrapper-padding-bottom;
+}
+
+.hc-form-field-flex {
+    display: inline-flex;
+    align-items: baseline;
+    box-sizing: border-box;
+    width: 100%;
+
+    padding: 0 7px 0 0;
+    border: 1.5px solid $gray-300;
+}
+
+.hc-form-field-prefix,
```

Comment:
Here they are! That doesn't match the css class listed in the other files...

Reply:
This refers to something else